### ISSUE TITLE
Camera Input

### ISSUE THREAD

**Issue by iGavroche** [TrinityEmulator/TrinityEmulator#7]
Trinity Emulator is amazing, and would greatly benefit from Native Camera Input support. Is it planned?
If not, are there resources you have gathered I could look into for helping push a PR towards this goal?

Thanks,
Gav

**Comment by andylin-hao** [comment]
Yes, we have implemented camera support for Trinity and planned to release it in May. Thanks for taking a interesting in it. Would very much appreciate your feedback then!
